EasyPost.Tests.Net/InsuranceTest.cs

```diff
@@ -1,145 +1,66 @@
-                {
-                    "street2", "Unit 1"
-                },
-                {
-                    "city", "San Francisco"
-                },
-                {
-                    "state", "CA"
-                },
-                {
-                    "zip", "94107"
-                },
-                {
-                    "country", "US"
-                }
-            };
-            ToAddressData = new Dictionary<string, object>()
-            {
-                {
-                    "company", "The White House"
-                },
-                {
-                    "street1", "1600 Pennsylvania Avenue NW"
-                },
-                {
-                    "street2", "DC 20500"
-                },
-                {
-                    "city", "Washington"
-                },
-                {
-                    "state", "DC"
-                },
-                {
-                    "zip", "20500"
-                },
-                {
-                    "country", "US"
-                }
-            };
-            ParcelData = new Dictionary<string, object>
-            {
-                {
-                    "weight", 8.0
-                }
-            };
+            VCR.SetUp(VCRApiKey.Test, "insurance", true);
+        }
 
-            Shipment shipment = Shipment.Create(new Dictionary<string, object>
-            {
-                {
-                    "from_address", FromAddressData
-                },
-                {
-                    "to_address", ToAddressData
-                },
-                {
-                    "parcel", ParcelData
-                },
-                {
-                    "service", "First"
-                },
-                {
-                    "carrier_accounts", new List<string>
-                    {
-                        {
-                            "ca_7642d249fdcf47bcb5da9ea34c96dfcf"
-                        }
-                    }
-                }
-            });
+        private static Insurance CreateBasicInsurance()
+        {
+            return Insurance.Create(Fixture.BasicInsurance);
+        }
 
-            InsuranceData = new Dictionary<string, object>
-            {
-                {
-                    "to_address", ToAddressData
-                },
-                {
-                    "from_address", FromAddressData
-                },
-                {
-                    "tracking_code", shipment.tracking_code
-                },
-                {
-                    "amount", "100.00"
-                },
-                {
-                    "carrier", "USPS"
-                }
-            };
+        [TestMethod]
+        public void TestCreate()
+        {
+            VCR.Replay("create");
+
+            Insurance insurance = CreateBasicInsurance();
+
+            Assert.IsInstanceOfType(insurance, typeof(Insurance));
+            Assert.IsTrue(insurance.id.StartsWith("ins_"));
```

Comment:
missing another assertion here, you should check the insurance amount is equal to 100

Reply:
Resolved in ef038dc01e2831d2938035fa0ff2541909a4dd5f